### DISCUSSION TITLE
Moving extra files?

### DISCUSSION THREAD

**Discussion by cwaddilove** [RandomNinjaAtk/docker-lidarr-extended#62] [Q&A] [answered]
Hello!

Do any of the additional scripts included with lidarr-extended take care of the years old issue of moving additional files with the music? (Cue, log, JPG/PNG, etc.)

**Comment by RandomNinjaAtk** [comment, marked as answer]
Its not really possible since Lidarr handles the import process. However, the script does extract the artwork and lyrc files from the songs embedded metadata to place them in the imported album folder accordingly... So if those were embedded into the files, it could be possible to do something like that... but still not ideal...
